Sources/JWTKit/ECDSA/ECDSAKey.swift

```diff
@@ -1,37 +1,11 @@
 @_implementationOnly import CJWTKitBoringSSL
 
 public final class ECDSAKey: OpenSSLKey {
-    public enum Curve: String, Codable {
-        case p256 = "P-256"
-        case p384 = "P-384"
-        case p521 = "P-521"
-
-        var cName: Int32 {
-            switch self {
-            case .p256:
-                return NID_X9_62_prime256v1
-            case .p384:
-                return NID_secp384r1
-            case .p521:
-                return NID_secp521r1
-            }
-        }
-      
-        init?(cName: Int32) {
-            switch cName {
-            case NID_X9_62_prime256v1:
-                self = .p256
-            case NID_secp384r1:
-                self = .p384
-            case NID_secp521r1:
-                self = .p521
-            default:
-                return nil
-            }
-        }
-    }
+	
+	@available(*, deprecated, renamed: "JWK.Curve")
+	public typealias Curve = JWK.Curve
     
-    public static func generate(curve: Curve = .p521) throws -> ECDSAKey {
+	public static func generate(curve: JWK.Curve = .p521) throws -> ECDSAKey {
```

Comment:
There looks to be some weird indentation here

Reply:
Seems my Xcode setup uses tabs 🤷🏼. Replaced all in https://github.com/vapor/jwt-kit/pull/82/commits/3c61d210de5a3be3a01965a4dd0e59923203d573 and https://github.com/vapor/jwt-kit/pull/82/commits/28a301a759f0b1a4b094b2c84c8866ce8924c569